aspnetcore/signalr/hubs.md

```diff
@@ -90,6 +90,22 @@ To make calls to specific clients, use the properties of the `Clients` object. I
 
 [!code-csharp[Send messages](hubs/sample/hubs/chathub.cs?range=15-24)]
 
+## Strongly typed Hubs
+
+The drawback with using `SendAsync` is that it relies on a magic string to provide the method to be called on the client. This leaves code open to runtime errors if the method name is misspelled or missing from the client.
```

Comment:
~~The drawback with~~
A drawback of

~~to provide the method to be called on the client~~
to specify the client method to be called